crates/solidity/testing/snapshots/cst_output/TryStatement/try_catch/input.sol

```diff
@@ -1 +1,3 @@
-try a.b() {} catch {}
\ No newline at end of file
+// Make sure that error recovery won't lead to misparsing
+// ambiguous function call options with the block following the try expression
```

Comment:
I suggest formatting the source code for readability, unless we are testing trivia specifically.

Reply:
related: the above syntax fails to parse. I suspect because it is not filtering trivia as needed:

```log
    Error: Expected Colon.
       ╭─[crates/solidity/testing/snapshots/cst_output/TryStatement/try_catch/input.sol:4:6]
       │
     4 │   bar();
       │      ──┬─  
       │        ╰─── Error occurred here.
    ───╯
  - >
    Error: Expected OpenBrace or ReturnsKeyword.
       ╭─[crates/solidity/testing/snapshots/cst_output/TryStatement/try_catch/input.sol:5:2]
       │
     5 │ ╭─▶ } catch {
     6 │ ├─▶ }
       │ │       
       │ ╰─────── Error occurred here.
    ───╯
```

Reply:
*sighs* that's quite an oversight, we do need to check for trivia tokens there. Done in https://github.com/NomicFoundation/slang/pull/862/commits/56a8cfa7bdc4c3ed12ec376bde73fbbb9dc8bfcd and https://github.com/NomicFoundation/slang/pull/862/commits/a9963df728065ad2d7033bed89271ab3f9e02d11; the former should help with #737 and the remaining PG clean up.